test/metabase/fingerprinting_test.clj

```diff
@@ -0,0 +1,145 @@
+(ns metabase.fingerprinting-test
+  (:require (clj-time [coerce :as t.coerce]
+                      [core :as t])
```

Comment:
we've generally been following doing requires with prefixes using vectors, e.g.

```clojure
[clj-time
 [coerce :as t.coerce]
 [core :as t]]
```

these days. If you install `clj-refactor` in Emacs you can use `M-x cljr-clean-ns` to automatically sort and clean the `:require` form for you